dw-snackbar.js

```diff
@@ -284,6 +284,16 @@ export class DwSnackbar extends layoutMixin(LitElement) {
     config.actionButton.callback && config.actionButton.callback(config.id);
   }
 
+  /**
+   * Purpose: To override the timeout at the base of toast data on the application.
```

Comment:
@shakil-mansuri-dream change java doc to:

`Returns timeout for the given toastId. It can be overridden to customize the behavior. e.g. To return the duration based on application requirement like N seconds for the Toasts without action buttons and M seconds for the Toasts with Action Buttons.`